### PR TITLE
add dumpling parameter

### PR DESCRIPTION
**Pull request by pepezzzz**
### First-time contributors' checklist 

- [X] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

expand dumpling example parameter 

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dumpling-overview.md [hunk 1]
```diff
@@ -87,12 +87,13 @@ dumpling -u root -P 4000 -h 127.0.0.1 --filetype sql -t 8 -o /tmp/test -r 200000
 {{< copyable "shell-regular" >}}
 
 ```shell
-./dumpling -u root -P 4000 -h 127.0.0.1 -o /tmp/test --filetype csv --sql 'select * from `test`.`sbtest1` where id < 100'
+./dumpling -u root -P 4000 -h 127.0.0.1 -o /tmp/test --filetype csv --sql 'select * from `test`.`sbtest1` where id < 100' -F 100MiB   --output-filename-template 'test.sbtest1.{{.Index}}'
```
Comment (ran-huang) [inline dumpling-overview.md:90]:
```suggestion
./dumpling -u root -P 4000 -h 127.0.0.1 -o /tmp/test --filetype csv --sql 'select * from `test`.`sbtest1` where id < 100' -F 100MiB --output-filename-template 'test.sbtest1.{{.Index}}'
```

Comment (ran-huang) [reply dumpling-overview.md:90]: `-F 100MiB --output-filename-template 'test.sbtest1.{{.Index}}'` 的含义能否加一下描述？

Comment (pepezzzz) [reply dumpling-overview.md:90]: -F 100MiB 是指定单个 csv 文件的大小
--output-filename-template 是指定 csv 文件名格式，如果不指定，会得到  results.{{.Index}}。

File: dumpling-overview.md [hunk 2]
```diff
@@ -87,12 +87,13 @@ dumpling -u root -P 4000 -h 127.0.0.1 --filetype sql -t 8 -o /tmp/test -r 200000
...
 
 ```shell
-./dumpling -u root -P 4000 -h 127.0.0.1 -o /tmp/test --filetype csv --sql 'select * from `test`.`sbtest1` where id < 100'
+./dumpling -u root -P 4000 -h 127.0.0.1 -o /tmp/test --filetype csv --sql 'select * from `test`.`sbtest1` where id < 100' -F 100MiB --output-filename-template 'test.sbtest1.{{.Index}}'
 ```
 
 > **注意：**
 >
 > - `--sql` 选项仅仅可用于导出 CSV 的场景。
+> - `--csv-separator`、`--csv-delimiter` 等选项可用于配置 CSV 文件的格式。
```
Comment (lichunzhu) [inline dumpling-overview.md:96]: 这个指令里没有体现这两项，要不命令里加一下？

File: dumpling-overview.md [hunk 3]
```diff
@@ -87,12 +87,13 @@ dumpling -u root -P 4000 -h 127.0.0.1 --filetype sql -t 8 -o /tmp/test -r 200000
...
 > - 该命令将在要导出的所有表上执行 `select * from <table-name> where id < 100` 语句。如果部分表没有指定的字段，那么导出会失败。
```
Comment (lichunzhu) [inline dumpling-overview.md:97]: 可以加上对 --output-filename-template 'test.sbtest1.{{.Index}} 的解释
```suggestion
> - 该命令将在要导出的所有表上执行 `select * from <table-name> where id < 100` 语句。如果部分表没有指定的字段，那么导出会失败。
> - 使用 `--sql` 配置导出时，dumpling 无法获知导出的表库信息，`--output-filename-template='test.sbtest1.{{.Index}}'` 选项可以配置导出的 csv 文件为 `test.sbtest1.000000000`, `test.sbtest1.000000001`... 方便后续 Lightning 导入。
```

File: dumpling-overview.md (hunk 2 above)
Comment (ran-huang) [reply dumpling-overview.md:96]: 文档末尾的 `dumpling-主要选项表`里有这两个参数的用法，可以链接过去。现在这个命令太长了，不建议再加

**Review by ran-huang — APPROVED** [review]

**Comment by ran-huang** [comment]
@pepezzzz @lichunzhu I made some changes to format and wording. PTAL.

**Review by lichunzhu — APPROVED** [review]
LGTM

**Comment by pepezzzz** [comment]
LGTM

**Pull request merged**